src/routes/teams.rs

```diff
@@ -30,21 +30,28 @@ pub async fn team_members_get(
                     user_id: data.user_id.into(),
                     role: data.role,
                     permissions: Some(data.permissions),
+                    accepted: data.accepted,
                 })
                 .collect();
 
             return Ok(HttpResponse::Ok().json(team_members));
         }
     }
 
-    let team_members: Vec<crate::models::teams::TeamMember> = members_data
-        .into_iter()
-        .map(|data| crate::models::teams::TeamMember {
```

Comment:
You could just add `.filter(|m| team_member.accepted)` here and not have to change anything else

Reply:
Hmm okay, will do that in the future. I'm a bit used to Linq and that causing double iteration.